molecularnodes/nodes/style_density_iso_surface.py

```diff
@@ -508,8 +508,14 @@ def style_density_iso_surface_node_group():
     # node Curve to Mesh
     curve_to_mesh = style_density_iso_surface.nodes.new("GeometryNodeCurveToMesh")
     curve_to_mesh.name = "Curve to Mesh"
-    # Fill Caps
-    curve_to_mesh.inputs[2].default_value = False
+    if bpy.app.version >= (4, 5, 0):
+        # Scale
+        curve_to_mesh.inputs[2].default_value = 1.0
```

Comment:
Ideally if you could change these to use the identifier instead of the input index. Future-proofs against changes in node inputs that way.

Reply:
Which would defend us against the current issue where `False` is being converted to `0` for the node input for the radius